docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -95,30 +92,31 @@ Expect the following output:
 serviceaccount/yugabyte-platform-universe-management created
 ```
 
-You need to grant access to this ServiceAccount using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.
-Ensure that you have replaced the `namespace` from the commands with the correct namespace of the previously created ServiceAccount.
+The next step is to grant access to this service account using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. 
+
+The namespace in the following commands needs to be replaced with the correct namespace of the previously created service account.
 
 The tasks you can perform depend on your access level.
 
 **Global Admin** can grant broad cluster level admin access by executing the following command:
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-global-admin.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```

Comment:
`platform-global-admin.yaml` has a placeholder `<SA_NAMESPACE>`. So this updated sed command does not replace the placeholder anymore.

Either
1. Change the command to use the correct placeholder
2. Change the `platform-globle-admin.yaml` to use the correct placeholder

Reply:
Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.